spec/controllers/editorial/news_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+RSpec.describe Editorial::NewsController, type: :controller do
+
+  let!(:root_node) { Fabricate(:root_node) }
+  let!(:news) { Fabricate(:section, name: 'news') }
+  let!(:section_a) { Fabricate(:section) }
+  let!(:section_b) { Fabricate(:section) }
+  let!(:section_c) { Fabricate(:section) }
+  let!(:user) { Fabricate(:user, author_of: section_a, reviewer_of: section_b) }
+
+  describe 'GET #new' do
```

Comment:
You should probably check that the response is success (especially in conjunction with render_views. This way you're smoke testing the view too.

    describe 'GET #new' do
      subject { get :new }

      context 'for an authorised user' do
        before { sign_in(user) }
        it { is_expected.to be_success }
      end

      context 'for a user who is not authorised' do
        it { is_expected.to redirect_to(root_path) }
        it { is_expected.to set_flash[:alert].to('You are not authorized to access this page.') }
      end
    end

Don't worry about checking instance variable assignment. You can consider Rails controller tests to be integration tests for the most part.